lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -148,6 +148,8 @@ def __eq__(self, other):
 class NetworkAnnotation:
     """NetworkAnnotation."""
 
+    NETWORK_ANNOTATION_RESOURCE_KEY = "k8s.v1.cni.cncf.io/networks"
```

Comment:
LGTM. I suggest to update the library documentation and example by adding the support of NETWORK_ANNOTATION_RESOURCE_KEY configuration.

Reply:
It's not a config param. I just extracted the constant.